acceptance-test/user-pathways/happy-path/happy-path.test.js

```diff
@@ -136,7 +136,6 @@ describe.only('User path(s)', () => {
         });
         await clickSelector(page, PV_CONTACT_DETAILS_EMAIL_SAFE_OPTION);
         await clickSelector(page, CONTINUE_BUTTON);
-        await clickSelector(page, CONTINUE_BUTTON);
```

Comment:
Needed to remove last step from acceptance test to stop PDF functionality from being called. @sulthan-ahmed